### PR TITLE
Updating changelog

### PR DESCRIPTION
**Pull request by josenavas**
I realize that I've not been updating the changelog as I've been developing all the changes in this branch. I think I added the important bits. Please, comment if you feel if I forgot something important and/or if the given wording is not accurate. I always have problems on how the wording should go in the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ Version 0.2.0-dev (Changes since version 0.2.0 go here)
 * The raw files of a RawData can be now updated using the `qiita db update_raw_data` CLI command.
 * instrument_model is now a required prep template column for EBI submissions.
 * PostgreSQL 9.3.0 is now the minimum required version because we are using the SQL type JSON, included for first time in 9.3.0.
+* The objects `RawData`, `PreprocessedData` and `ProcessedData` have been removed from the system and substituted by a general `Artifact` object.
+* The CLI commands `load_raw`, `load_preprocessed` and `load_processed` have been removed from the system and substituted by `load_artifact`.
+* The processing of the artifacts have been moved to a new `target_gene` plugin using the new plugin system.
```
Comment (antgonza) [inline CHANGELOG.md]: Not sure if this is accurate. I guess a better description could be split in two:
- We incorporated the idea of plugins into the system. Now, all processing could be plugins.
- QIIME workflows for splitting libraries (SFF/FASTA-QUAL and FASTQ/per-sample-FASTQ) and for picking OTUs has been moved to a new target gene plugin.

**Comment by antgonza** [comment]
1 comment, after that :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (josenavas) [inline CHANGELOG.md]: Done

**Comment by squirrelo** [comment]
:+1:

**Pull request merged**